lang/fr_FR.json

```
@@ -0,0 +1,416 @@
+    "Comment must not be longer than 2,000 characters.": "Le commentaire ne doit pas dépasser 2 000 caractères.",
+    "Comments": "Commentaires",
+    "Comments - {{achievementTitle}}": "Commentaires - {{achievementTitle}}",
+    "Comments - {{gameTitle}}": "Commentaires - {{gameTitle}}",
+    "Comments - {{leaderboardTitle}}": "Commentaires - {{leaderboardTitle}}",
+    "Comments - {{user}}": "Commentaires - {{user}}",
+    "Comments on a forum topic I'm involved in": "Commentaires sur un sujet de forum auquel je participe",
+    "Comments on an achievement I created": "Commentaires sur un succès que j'ai créé",
+    "Comments on my activity": "Commentaires sur mon activité",
+    "Comments on my user wall": "Commentaires sur mon mur utilisateur",
+    "Comments: {{leaderboardTitle}}": "Commentaires : {{leaderboardTitle}}",
+    "Completed": "Terminé",
+    "Confirm New Email Address": "Confirmer la Nouvelle Adresse Email",
+    "Confirm Password": "Confirmer le Mot de Passe",
+    "Connect API Key": "Clé API Connect",
+    "Contact Us": "Contactez-nous",
+    "Content you wrote in forums, comments, etc. will NOT be removed.": "Le contenu que vous avez écrit dans les forums, commentaires, etc. ne sera PAS supprimé.",
+    "Copied your web API key!": "Votre clé API web a été copiée !",
+    "Copy to clipboard": "Copier dans le presse-papiers",
+    "Copyright": "Droits d'Auteur",
+    "Couldn't find any active players.": "Impossible de trouver des joueurs actifs.",
+    "Couldn't find any completed claims.": "Impossible de trouver des demandes complétées.",
+    "Couldn't find any sets in progress.": "Impossible de trouver des ensembles en cours.",
+    "Create Ticket": "Créer un Ticket",
+    "Current Email Address": "Adresse Email Actuelle",
+    "Current Locale": "Langue Actuelle",
+    "Current Password": "Mot de Passe Actuel",
+    "Currently Online": "Actuellement en ligne",
+    "Customize View": "Personnaliser la vue",
+    "Delete Account": "Supprimer le compte",
+    "Delete All Comments on My User Wall": "Supprimer tous les commentaires sur mon mur utilisateur",
+    "Delete comment": "Supprimer le commentaire",
+    "Deleted!": "Supprimé !",
+    "Deleting...": "Suppression...",
+    "Derived from points, number of achievers, and number of players.": "Dérivé des points, du nombre de réussites et du nombre de joueurs.",
+    "Descending (Z - A)": "Descendant (Z - A)",
+    "Details on how the hash is generated for each system can be found <1>here</1>.": "Les détails sur la génération de l'empreinte pour chaque système se trouvent <1>ici</1>.",
+    "Dev": "Dev",
+    "DevQuest": "DevQuest",
+    "Developer Compliance": "Conformité Développeur",
+    "Disclaimer about ROMs": "Avertissement concernant les ROMs",
+    "Disclaimers": "Avertissements",
+    "Documentation": "Documentation",
+    "Download Emulator": "Télécharger l'émulateur",
+    "Download Patch File": "Télécharger le fichier de correctif",
```

Les patchs sont principalement pour les hacks et subsets, rarement pour corriger des problèmes
```suggestion
    "Download Patch File": "Télécharger le fichier de patch",
```